src/System.Collections.Specialized/System.Collections.Specialized.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.30723.0
+# Visual Studio 14
```

Comment:
It shouldn't affect anyone and since we can run tests only from VS 2015 I think it is better to keep